controllers/front/successIFrame.php

```diff
@@ -62,18 +68,7 @@ public function postProcess() // todo refactor this by the logic provided
         try {
             Tools::redirect($this->getOrderConfirmationLink($cartId, $moduleId, $orderId, $secureKey));
         } catch (Exception $e) {
-            PrestaShopLogger::addLog(
-                sprintf(
-                    '%s has caught an error: %s',
-                    __CLASS__,
-                    $e->getMessage()
-                ),
-                1,
-                null,
-                null,
-                null,
-                true
-            );
+            $logger->error(sprintf('%s - %s', self::FILE_NAME, $e->getMessage()));
```

Comment:
$logger->error('Failed to validate security token', [
                'context' => [],
                'exceptions' => ExceptionUtility::getExceptions($exception),
            ]);